### PR TITLE
Add transcribe eventbridge

### PR DESCRIPTION
**Pull request by bjhogan**
*Description of changes:*

Adds Transcribe Job State Change Event. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Libraries/src/Amazon.Lambda.CloudWatchEvents/TranscribeEvents/TranscribeJobStateChange.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+namespace Amazon.Lambda.CloudWatchEvents.TranscribeEvents
+{
+    /// <summary>
+    /// This class represents the details of a Transcribe Job State Change sent via EventBridge.
```
Comment (normj) [inline Libraries/src/Amazon.Lambda.CloudWatchEvents/TranscribeEvents/TranscribeJobStateChange.cs:4]: Is the a URL you can add to where this event type is documented?

**Review by normj — CHANGES_REQUESTED** [review]
We need a unit test for this new POCO. You can look at the [CloudWatchEventsS3ObjectCreate](https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/test/EventsTests.Shared/EventTests.cs#L3023) test as an example.

File: Libraries/src/Amazon.Lambda.CloudWatchEvents/TranscribeEvents/TranscribeJobStateChange.cs (hunk 1 above)
Comment (bjhogan) [reply Libraries/src/Amazon.Lambda.CloudWatchEvents/TranscribeEvents/TranscribeJobStateChange.cs:4]: https://docs.aws.amazon.com/transcribe/latest/dg/monitoring-events.html

Comment (normj) [reply Libraries/src/Amazon.Lambda.CloudWatchEvents/TranscribeEvents/TranscribeJobStateChange.cs:4]: Sorry I wasn't clear, can you add the URL as part of the NDoc here?

**Review by normj — CHANGES_REQUESTED** [review]

**Review by normj — APPROVED** [review]

**Comment by normj** [comment]
PR has been released as part of version 4.2.0

**Pull request merged**